lib/logic/entities/Environment.js

```diff
@@ -0,0 +1,18 @@
+const Entity = require('./Entity');
+
+class Environment extends Entity {
+    constructor(data) {
+        super();
+        const info = {};
+        this.info = info;
+        this.entityType = 'environment';
+        this.info = info;
+        info.id = data._id;
+        info.status = data.creationStatus;
+        info.name = data.name;
+        this.defaultColumns = ['id', 'name'];
```

Comment:
please explain in the base entity what is the meaning of 'defaultColumns' vs 'wideColumns'
just so that the developer will understand